README.md

```diff
@@ -91,9 +91,25 @@ TODO: Include architectural explanation and diagram here.
 
    
 
-
 ### Notes
 
+**Shibboleth SP:**
+In a BU wordpress fargate stack, the shibboleth SP can reside in one of two places:
+
+1. Mod_shib: The traditional setup in which wordpress containers running apache would have mod_shib installed, and authentication with shibboleth would be carried out from within.
+2. A newer option is available to build the Fargate cluster such that the Wordpress containers running in it no longer involve themselves with being saml SP clients to shibboleth using the mod_shib module. A DNS option has been added to context parameters to opt for this situation be serviced by a pre-existing Cloudfront distribution that takes over this saml client role. Currently, this Cloudfront distribution is created in a separate CDK stack and it should be  configured to target the ALB of this stack, once it becomes available, as an origin. A lambda@edge origin request function within this separate stack processes all incoming requests and carries out the saml negotiations with the BU shibboleth IDP. This has a few implications:
```

Comment:
I think this means that if the context does not contain a property named `DNS`, then the cloudfront distribution and associated resources will not be deployed? If so, then maybe clarify that point a little as it seems important.